### ISSUE TITLE
fix(jans-casa): blank menu after logging in to user created via agama journey

### ISSUE THREAD

**Issue by SafinWasi** [JanssenProject/jans#6328]
**Describe the bug**
When logging in to a user enrolled via Agama Journey, the Casa home page is blank, with no menus.

**To Reproduce**
1. Start by triggering the Agama Lab Journey main flow. For this test, the flow was started by Jans Tarp.
2. Enroll credentials to complete creating the user
3. After the redirect back to RP, visit `/jans-casa` to access the Casa dashboard. Note that you must close the Jans Tarp extension tab before this step, otherwise the Casa tab will auto close.

**Expected behavior**
The Casa dashboard should have menus to modify MFA credentials.

**Screenshots**
![image](https://github.com/JanssenProject/jans/assets/6601566/c8e8f61c-fd53-4e85-8a16-8250fee3de11)

**Desktop (please complete the following information):**
  - OS: Arch Linux kernel 6.5.7
  - Browser: Firefox 
  - Version : 118.0.2

**Comment by SafinWasi** [comment]
`casa.log` with debug during login:
```
18-10 20:32:24.230 INFO  [qtp1897871865-18] ui.vm.admin.LogLevelViewModel LogLevelViewModel.java:50- Log level changed to DEBUG
18-10 20:32:46.135 INFO  [qtp1897871865-17] casa.core.filter.LocaleInterceptor LocaleInterceptor.java:43- Browser locale is 'en_US'
18-10 20:32:46.135 INFO  [qtp1897871865-17] casa.core.filter.LocaleInterceptor LocaleInterceptor.java:54- Locale for this session will be 'en'
18-10 20:32:46.136 DEBUG [qtp1897871865-17] casa.core.navigation.HomeInitiator HomeInitiator.java:99- Starting authorization flow
18-10 20:32:46.137 DEBUG [qtp1897871865-17] jans.casa.misc.WebUtils WebUtils.java:103- Redirecting to URL=https://jans.test/jans-auth/restv1/authorize?scope=openid+profile+user_name+clientinfo&acr_values=casa&response_type=code&redirect_uri=https%3A%2F%2Fjans.test%2Fjans-casa&state=qj7XY514nbmiGRoKMKHxzbb-rXquEFtttrCbveqlcdA&client_id=3000.b05ef8e0-ec18-4e87-b773-a4243393ead1
18-10 20:33:10.409 DEBUG [qtp1897871865-17] jans.casa.rest.AuthorizationProcessingFilter AuthorizationProcessingFilter.java:72- Validating token 19cfc622-f8ef-4e25-89f3-4f79e96a12e0
18-10 20:33:10.417 INFO  [qtp1897871865-17] jans.casa.rest.AuthorizationProcessingFilter AuthorizationProcessingFilter.java:90- Authorization passed
18-10 20:33:10.417 DEBUG [qtp1897871865-17] jans.casa.rest.NewSecondFactorUserWS NewSecondFactorUserWS.java:62- ACRs correlated with plugins or internal extensions: [super_gluu, otp, fido2, twilio_sms, smpp]
18-10 20:33:10.420 DEBUG [qtp1897871865-18] jans.casa.rest.AuthorizationProcessingFilter AuthorizationProcessingFilter.java:72- Validating token 19cfc622-f8ef-4e25-89f3-4f79e96a12e0
18-10 20:33:10.426 INFO  [qtp1897871865-18] jans.casa.rest.AuthorizationProcessingFilter AuthorizationProcessingFilter.java:90- Authorization passed
18-10 20:33:23.920 DEBUG [qtp1897871865-21] casa.core.navigation.HomeInitiator HomeInitiator.java:99- Starting authorization flow
18-10 20:33:23.921 DEBUG [qtp1897871865-21] jans.casa.misc.WebUtils WebUtils.java:103- Redirecting to URL=https://jans.test/jans-auth/restv1/authorize?scope=openid+profile+user_name+clientinfo&acr_values=casa&response_type=code&redirect_uri=https%3A%2F%2Fjans.test%2Fjans-casa&state=GRy6dc5gBCUetAbstPx64p-A_BTBTmcK-_e5HGXA1OE&client_id=3000.b05ef8e0-ec18-4e87-b773-a4243393ead1
18-10 20:33:24.602 DEBUG [qtp1897871865-21] casa.core.navigation.HomeInitiator HomeInitiator.java:69- Adding user to session
18-10 20:33:24.787 DEBUG [qtp1897871865-18] casa.core.navigation.HomeInitiator HomeInitiator.java:79- Taking user to homepage...
18-10 20:33:24.787 DEBUG [qtp1897871865-18] jans.casa.misc.WebUtils WebUtils.java:103- Redirecting to URL=user.zul
```

**Comment by jgomer2001** [comment]
fixed via #6341
